node-repository/src/main/java/com/yahoo/vespa/hosted/provision/node/IP.java

```diff
@@ -325,10 +325,14 @@ public int eventuallyUnusedAddressCount(NodeList nodes) {
             return ipAddresses.protocol;
         }
 
-        public Set<String> getIpSet() {
+        public Set<String> asSet() {
             return ipAddresses.asSet();
         }
 
+        public boolean isEmpty() {
```

Comment:
The method name is misleading as it indicates there are no addresses in this pool either.

Reply:
Ok, I'll revert. It's very confusing that an _IP address pool_ contains something that's not an IP address, which is also called "Address".